install-build-deps.sh

```diff
@@ -13,15 +13,15 @@ fi
 # Install dependencies for openSUSE
 if [ -e /usr/bin/zypper ] ; then
     sudo zypper refresh
-    sudo zypper in -y build git-core gcc wget make glibc-devel glib2-devel libarchive-devel \
+    sudo zypper in -y build git-core gcc g++ wget make glibc-devel glib2-devel libarchive-devel \
```

Comment:
What introduces this dependency?

Reply:
I don't remember, actually. I think it was necessary in order to be able to build CMake, which is no longer necessary. I'll change this back in the post-CMake fixes mentioned below.

Reply:
As said, I'll add it to the TODO list, with low priority.